contributing.md

```diff
@@ -0,0 +1,146 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Filing an issue
+
+1. Look for a similar issue that may already describe the proposed changes.
+
+1. If none exists, create a new issue and describe the proposed changes.
+
+1. Apply any relevant labels to the issue and set the milestone to Backlog.
+
+## Pull Request Process
+
+1. Find the issue that describes the changes you are going to make.  If no issue exists
+for the changes you want to make, see above and file an issue describing the changes, then
```

Comment:
The issue shouldn't describe the changes. The pull request description should do that. However, the issue should describe the problem addressed.

Reply:
@peterbe I think you're conflating change with implementation?

Reply:
I don't understand that reply. The sentence is "file an issue describing the changes". That to me sounds like "file an issue describing the implementation changes to be". I.e. the *issue* description talking about what's *going to change* or what's *going to be implemented*. 

It's hard to argue on English'y stuff for a non-native speaker, as I always assume that I'm most likely wrong, but my intention to say is this:

* GOOD issue description: *"Clicking the Save button causes an error."*
* BAD issue description: *"Prevent the Save button being clickable if [condition]"*
* GOOD Pull Request description: *"Prevent the Save button being clickable if [condition]"*
* BAD Pull Request description: *"Clicking the Save button causes an error."*

I'm definitely not going to hold up *this* PR. These are merely my reactions and 2-cents jotted down from reading the proposed new text.

Reply:
@peterbe Okay I'll **_change_** the wording ;)